docs/notebooks/Notebook_01_buildH_calc_OP.ipynb

```diff
@@ -280,32 +291,45 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 8,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "Constructing the system...\n",
+      "/home/pierre/.soft/miniconda3/envs/buildh/lib/python3.9/site-packages/MDAnalysis/topology/base.py:203: DeprecationWarning: `np.int` is a deprecated alias for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.\n",
+      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
+      "  residx = np.zeros_like(criteria[0], dtype=np.int)\n",
+      "/home/pierre/.soft/miniconda3/envs/buildh/lib/python3.9/site-packages/MDAnalysis/core/selection.py:521: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.\n",
+      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
+      "  mask = np.zeros(len(group), dtype=np.bool)\n",
```

Comment:
Suggestion: delete by hand those #!*$ warnings

Reply:
The user might not understand why those error messages occur. So it's probably better to let them but adding an explanation that everything is under control.

Reply:
We already tell the user here about the warnings: https://buildh.readthedocs.io/en/latest/usage.html#numpy-warnings
To me that's enough.

Reply:
I would remove those also since it's occurs only for numpy 1.20

Reply:
So @pierrepo, do you agree to remove those warnings?

Reply:
I would disagree for two reasons:
1. The user might think there is a real issue with buildH if the output in the notebook differs from the one he/she could experience in his/her terminal.
2. It implies to modify manually the json in the notebook that is usually not a good idea.

Could be try updating [MDAnalysis to 1.1.1](https://github.com/MDAnalysis/mdanalysis/releases)? It looks like some Numpy deprecation messages have been handled?

Reply:
> Could we try updating [MDAnalysis to 1.1.1](https://github.com/MDAnalysis/mdanalysis/releases)? It looks like some Numpy deprecation messages have been handled.

Yes, very good idea! Maybe we can create another issue for that, so I can merge this PR which is about the `-v` option?

Reply:
Yes. We might have to rebuild the notebooks anyway if we upgrade buildH version.